packages/workbox-precaching/utils/deleteOutdatedCaches.mjs

```diff
@@ -28,7 +28,7 @@ const SUBSTRING_TO_FIND = '-precache-';
  * @private
  * @memberof module:workbox-precaching
  */
-export async function cleanupOutdatedCaches(currentPrecacheName,
+export async function deleteOutdatedCaches(currentPrecacheName,
```

Comment:
Do you want to convert this to `const deleteOutdatedCaches = (...) => {}` too? It looks like this PR includes a number of similar conversions.

Reply:
Sure.

BTW, I've personally always preferred inline `export` declarations, but I discovered this weekend that [they don't place nicely with jsdocs](https://github.com/jsdoc3/jsdoc/issues/1272), so I think it makes sense to always use an export block at the bottom of a file.

Reply:
Ahh, now I see you were talking more about function declarations vs assigning an arrow function to a const.